namespaces/live-1.cloud-platform.service.justice.gov.uk/claim-criminal-injuries-compensation-staging/resources/rds.tf

```diff
@@ -0,0 +1,37 @@
+variable "cluster_name" {}
+variable "cluster_state_bucket" {}
+
+module "rds" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=4.1"
```

Comment:
Hi. Can you please upgrade to the latest version of the RDS module. 
https://github.com/ministryofjustice/cloud-platform-terraform-rds-instance/releases